### ISSUE TITLE
[UserStory] Display prompts before requesting input when running demo through ant

### ISSUE THREAD

**Issue by ashleydanie** [MetroCS/redistricting#98]
# User Story
### Essential components
- [x] Title describes the story
- [x] Stakeholder type is identified
- [x] Outcome is described
- [x] Rationale is explicit
- [x] Acceptance criteria are verifiable and from the perspective of the stakeholder

## Story
**As a**  _Maintainer_  
**I want**  _The prompts displayed before entering input when I run the demo through the terminal using "ant run"_
**so that**  _I can use the demo properly_

## Acceptance Criteria   
**when**  _running demo in terminal using "ant run"_  
**then**  _the demo should display prompts for the input_  

## Additional Information
_Currently there is an issue where the prompts will show only after inputs have been entered_

**Comment by jody** [comment]
Some thoughts on this...
* The demonstration prototype appears to run as expected from the command line.
* This issue might be addressed by
  * removing the Run target from build.xml
  * replacing the Run target with instructions for how to run the demo from the command line
  * adding prefacing instructions in the Run target before executing the java command 


Related issue #90

**Comment by jody** [comment]
Removing RedistrictingDriver from the package is becoming a highly likely outcome. All issues/PRs that address concerns involving RedistrictingDriver are thus likely to be irrelevant. Am putting all such activity "on hold" for now.
